### DISCUSSION TITLE
Scrape game names

### DISCUSSION THREAD

**Discussion by dirn** [OnionUI/Onion#1289] [Ideas]
In addition to images, Skraper pulls down metadata about each game. Most of this data is ignored but the game's name is displayed. This is a nice quality of life improvement over viewing the ROM names.

    Mike Tyson's Punch-Out!! (Japan, USA) (En) (Rev 1)

versus

    Mike Tyson's Punch-Out!!

I tried to switch to the on-device scraper. It didn't create a `miyoogamelist.xml`; it only downloaded the images. Would it be possible to add an option to generate/update `miyoogamelist.xml`?

**Comment by Phanturian** [comment]
In Apps>Tweaks>Tools there is an option for Generate miyoogamelist with digest names. You can press select while highlighting this option for a detailed description. I haven't tried it, but I think it's what you're looking for.

**Reply by dirn** [reply]
Well that certainly seemed to do the trick. It’s a bit heavy handed for times when I’m only adding one game, but it’s better than having nothing at all. Thanks for pointing that out!

**Comment by schmurtzm** [comment]
Tweaks already allows to make what you want to do. However I keep the idea in mind for Scraper too ;).
